src/webview_provider/docsEditPanel.ts

```diff
@@ -389,6 +389,7 @@ export class DocsEditViewPanel implements WebviewViewProvider {
       dbtVersion &&
       dbtVersion[0] >= 1 &&
       dbtVersion[1] >= 8 &&
```

Comment:
Please use semver instead this comparisons

Reply:
done